server/src/validators/supplierValidator.ts

```diff
@@ -0,0 +1,26 @@
+import { Validator } from "../types";
+
+const supplierCreateValidator: Validator[] = [
```

Comment:
You should add a supplierUpdateValidator where all of the fields are not required so the validator function will work